src/components/StackConfigurator.tsx

```diff
@@ -69,6 +69,18 @@ export const StackConfigurator = forwardRef(function StackConfigurator(
     const { data: groups, isLoading: isLoadingGroups } = useApi<Group[]>(getGroups)
     const [group, setGroup] = useState('')
 
+    const ttlMap = useMemo(() => new Map<string, number>([
+        ["1 hour", 60 * 60],
+        ["6 hours", 60 * 60 * 6],
+        ["12 hours", 60 * 60 * 12],
+        ["1 day", 60 * 60 * 24],
+        ["2 days", 60 * 60 * 24 * 2],
+        ["5 days", 60 * 60 * 24 * 5],
+        ["1 week", 60 * 60 * 24 * 7],
+        ["2 weeks", 60 * 60 * 24 * 7 * 2],
+        ["1 month", 60 * 60 * 24 * 7 * 4],
+    ]), [])
```

Comment:
This is purely a static constant so should be created outside of the function body. That way you also don't need `useMemo`. You would use `useMemo` if you want to create a memoized variable which is based on another variable which is only available within the function body, typically some prop or state variable.